interface/wx/webrequest.h

```diff
@@ -422,6 +422,26 @@ class wxWebRequest
     */
     void SetStorage(Storage storage);
 
+    /**
+        Sets the timeouts in milliseconds for the webrequest.
```

Comment:
```suggestion
        Set the timeouts for the connection and total request time.
```

(the difference is "Set" instead of "Sets", for consistency with the other functions, and I think the fact that "ms" is mentioned in the parameters descriptions is enough).